### DISCUSSION TITLE
Unable to create core in Apache solr thats why I am unable to connect Drupal app with Apache solr

### DISCUSSION THREAD

**Discussion by soumyajit40cs** [docksal/docksal#1832] [General]
Unable to create core in Apache solr thats why I am unable to connect Drupal app with Apache solr

**Comment by gangaftagley** [comment]
Not very helpful error message - but I was having the same issue yesterday - updated docker desktop to latest version and used solr:9.7 as the solr image and it finally worked (MacBook Pro M3 pro chip)
